code-quality/rules/pmd/ruleset.xml

```diff
@@ -30,9 +30,7 @@
   <rule ref="rulesets/java/unnecessary.xml">
     <exclude name="UselessParentheses" />
   </rule>
-  <rule ref="rulesets/java/unusedcode.xml">
-    <exclude name="UnusedModifier" />
```

Comment:
For some reason, pmd started complaining about this rule not being available so I had to remove it.
This PR doesn't touch anything about the CI its static analysis rules . Am I missing something here?